### PR TITLE
Fixed red flag splitting across two lines

### PR DESCRIPTION
**Pull request by michaelspellacy**
Addressed Issue #41 Added white-space: nowrap to .flag_red and also added new properties to link within toc to compensate for nowrap. Tested across several browsers and devices. https://www.screencast.com/t/zAm8KP7CPciC Hope this helps! 

p.s. Love what you are doing here!

### CODE REVIEW

File: assets/css/structures/_table-of-content.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
 .toc__list-item {
   break-inside: avoid;
+
+    a {
+    display: inline-block;
+    margin: 0.5rem 0.75rem;
```
Comment (tatianamac) [inline assets/css/structures/_table-of-content.scss]: This `margin` seems to be adding some extra spacing around each linked word. Is this visual adjustment necessary for the red flags to not break?

If it's not necessarily, I'd prefer the alignment to be left-aligned without the extra padding.

_Left is original where all text is aligned left_; right is PR where there is extra spacing

Side-by-side of [1] correctly rendered text (without a leading space) and [2] incorrectly rendered text (with a leading space)

File: assets/css/base.scss [hunk 2]
```diff
@@ -419,12 +419,13 @@ th {
 
 .flag__red {
   background-color: rgb(255, 192, 203);
+  border-radius: 1rem;
   font-size: 0.9rem;
   font-weight: bold;
-  border-radius: 1rem;
-  padding: 0.45rem 0.65rem;
   margin: 0.25rem 0.75rem;
+  padding: 0.45rem 0.65rem;
```
Comment (tatianamac) [inline assets/css/base.scss]: More a curiosity than an issue if you don't mind explaining (for my own edification): Why did you rearrange these properties? 🤓

File: assets/css/structures/_table-of-content.scss [hunk 3]
```diff
@@ -1,3 +1,9 @@
 .toc__list-item {
   break-inside: avoid;
+
+    a {
+    display: inline-block;
```
Comment (tatianamac) [inline assets/css/structures/_table-of-content.scss]: The `inline-block` here causes an issue with the underline for words that become multiple lines, like OCD:

Obsessive Compulsive Disorder has underline that extends beyond the word

File: assets/css/structures/_table-of-content.scss (hunk 1 above)
Comment (ovlb) [reply assets/css/structures/_table-of-content.scss]: I think it’s fine to have `margin-left: 0` and go with the slightly indented flag. Which happens with non defined words, too.

**Comment by tatianamac** [comment]
@michaelspellacy Are you still able to address these comments? No rush on our end as I imagine you're busy, but want to check in. :)

File: assets/css/base.scss (hunk 2 above)
Comment (michaelspellacy) [reply assets/css/base.scss]: @tatianamac  Sorry again for delay! Day job got out of hand! I simply reorder them alphabetically out of habit for better readability, but happy to change them back in preference to what you may already have in place throughout CSS.

Comment (tatianamac) [reply assets/css/base.scss]: No sorries. Understood; carry on as you are. I was mostly curious.

**Comment by michaelspellacy** [comment]
Hi @tatianamac and @ovlb!

Took a poke at revisiting this. You'd think it was easy, but no, it never is. :-) This is what I did:

**eleventy.js** - I really felt that a new span was necessary to get us the hook we needed to balance styles on terms with and without hyperlinks. 

**_flag.scss** - Restored old order. Adjusted margin. Added display: inline-block;

**_table-of-contents.scss** - Included <a> and <span> elements (feel free to add a shared class to these in the future). Added margin-right: 0.75rem since it was removed from left margin of .flag__red

Think it looks much better now. Let me know if any new issues!

**Comment by tatianamac** [comment]
@ovlb Since you updated the sass last, do you mind taking a look over this? I'll close it once you give it 🐚 of approval

**Comment by ovlb** [comment]
With thanks to @michaelspellacy my sea( shel)l of approval is given. 🐚

**Comment by michaelspellacy** [comment]
Thanks gang! Glad I could help! ❤️

**Pull request merged**